halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/providers/aws/AwsAccount.java

```diff
@@ -17,24 +17,29 @@
 
 package com.netflix.spinnaker.halyard.config.model.v1.providers.aws;
 
+import com.amazonaws.auth.AWSCredentials;
 import com.netflix.spinnaker.halyard.config.model.v1.node.Account;
 import com.netflix.spinnaker.halyard.config.model.v1.node.Validator;
 import com.netflix.spinnaker.halyard.config.problem.v1.ConfigProblemSetBuilder;
 import lombok.Data;
 import lombok.EqualsAndHashCode;
+import lombok.ToString;
 
 import java.util.ArrayList;
 import java.util.List;
 
 @Data
 @EqualsAndHashCode(callSuper = true)
+@ToString(exclude = {"awsCredentials"})
 public class AwsAccount extends Account {
 
   private String defaultKeyPair;
   private String edda;
   private String discovery;
   private String accountId;
   private List<AwsRegion> regions = new ArrayList<>();
+  private String assumeRole;
+  private AWSCredentials awsCredentials;
```

Comment:
I don't think this should appear here, I don't know how it's serialized & this should be easily editable via yaml